curp/src/cmd.rs

```diff
@@ -19,6 +19,9 @@ pub trait Command:
     /// Get keys of the command
     fn keys(&self) -> &[Self::K];
 
+    /// Get cmd id
+    fn id(&self) -> &CmdId;
```

Comment:
Is this something like propose id?

Reply:
It's propose id actually, let me rename it.